src/recipe/Baked.Recipe.Service.Application/CodingStyle/RichTransient/LookUpTransientByIdConvention.cs

```diff
@@ -0,0 +1,29 @@
+﻿using Baked.Business;
+using Baked.Domain.Model;
+using Baked.RestApi.Configuration;
+
+namespace Baked.CodingStyle.RichTransient;
+
+public class LookUpTransientByIdConvention(DomainModel _domain)
+    : IApiModelConvention<ParameterModelContext>
+{
+    public void Apply(ParameterModelContext context)
+    {
+        if (context.Action.MappedMethod is null) { return; }
+        if (context.Action.MappedMethod.Has<InitializerAttribute>()) { return; }
+        if (!context.Parameter.IsInvokeMethodParameter) { return; }
+        if (context.Parameter.MappedParameter is null) { return; }
+        if (!context.Parameter.MappedParameter.ParameterType.TryGetMembers(out var members)) { return; }
+        if (!members.Has<LocatableAttribute>()) { return; }
+        if (members.TryGetQueryType(_domain, out var _)) { return; }
```

Comment:
this creates a dependency to orm feature. don't use orm or data access or related coding style extensions.